packages/ui/src/camel-utils/camel-to-table.adapter.ts

```diff
@@ -15,6 +15,32 @@ export interface IPropertiesTableFilter<T> {
   filterValue: T[keyof T];
 }
 
+/**
+ * Get class name from fully qualified name
+ * e.g.
+ * String => Strin
+ * java.util.Set<java.nio.file.OpenOption> => Set
+ * org.apache.camel.component.as2.AS2Component => AS2Component
+ * org.hl7.fhir.instance.model.api.IPrimitiveType<java.util.Date> => IPrimitiveType
+ * @param fullyQualifiedName - Fully qualified name
+ * @returns Class name only
+ */
+const getClassNameOnly = (fullyQualifiedName: string): string => {
+  if (fullyQualifiedName.match('^[A-Za-z0-9]+$')) {
```

Comment:
Since creating `RegExp` object is expensive in Javascript, could we move both `RegExp`s to the top of the file and reuse them from there?

Reply:
sure, fixed